gateway/src/integration-test/java/com/hurence/util/AssertResponseGivenRequestHelper.java

```diff
@@ -71,5 +71,22 @@ public void assertWrongRequestGiveResponseFromFile(Vertx vertx, VertxTestContext
                     });
                 }));
     }
+    public void assertMissingPointsRequestGiveArrayResponseFromFile(Vertx vertx, VertxTestContext testContext,
+                                                       String requestFile, String responseFile) {
+        final FileSystem fs = vertx.fileSystem();
+        Buffer requestBuffer = fs.readFileBlocking(AssertResponseGivenRequestHelper.class.getResource(requestFile).getFile());
+        webClient.post(endpoint)
+                .as(BodyCodec.jsonObject())
+                .sendBuffer(requestBuffer.getDelegate(), testContext.succeeding(rsp -> {
+                    testContext.verify(() -> {
+                        assertEquals(207, rsp.statusCode());
```

Comment:
I think code 201 is more appropriate here. Code 207 seems to be for webDAV servers, Anyway use the static variable you defined instead of a hard coded value.

Reply:
done